src/js/Core.js

```diff
@@ -220,9 +221,10 @@ Wui.positionItem = function(parent,child){
                     })(),
         plRight =   (ofst.left + parent.outerWidth() - cWidth > 0);
 
-    // If the parent is not a fixed-position element, then add the scrollTop in case the page is scrolled down.
-    if(fxdOrAbs === 'fixed')
+    // If we we are not in a dialog window then add the scrollTop in case they have scrolled down.
+    if(win.length == 0) {
```

Comment:
Use `===` and `!==` over `==` and `!=` when comparing with 0, 1, true, false, undefined, or null. ([Style Guide rule 6.0](http://www.wui-js.com/js-style-guide.html#6_0))